content/pt/community/static/cncf-code-of-conduct.md

```diff
@@ -0,0 +1,44 @@
+<!-- Do not edit this file directly. Get the latest from
+     https://github.com/cncf/foundation/blob/master/code-of-conduct.md -->
```

Comment:
As stated by this line, this file should be an **exact** copy of the original CNCF CoC file in pt:

https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/pt.md

If this updated version is an improvement, feel free to PR the https://github.com/cncf/foundation repo and update the original file. Then copy the updated file to this repo. Check  https://github.com/cncf/foundation/pull/35 for example.

Reply:
I'm sorry. Changing to the correct one.